jina/peapods/runtimes/remote/__init__.py

```diff
@@ -0,0 +1,82 @@
+import argparse
+from typing import Union, Dict
+
+from jina.peapods.peas import PeaRoleType
+from jina.peapods.zmq import Zmqlet, send_ctrl_message
+from jina.peapods.runtimes import BaseRuntime
+
+__all__ = ['RemoteRuntime']
+
+
+class RemoteRuntime(BaseRuntime):
```

Comment:
It is not a runnable Class
```suggestion
class BaseRemoteRuntime(BaseRuntime):
```